tools/ide_plugins/pysa-vscode/syntaxes/Pysa.tmLanguage

```diff
@@ -0,0 +1,4007 @@
+						</dict>
+					</dict>
+					<key>name</key>
+					<string>string.quoted.single.block.pysa</string>
+					<key>patterns</key>
+					<array>
+						<dict>
+							<key>begin</key>
+							<string>(?=(SELECT|INSERT|UPDATE|DELETE|CREATE|REPLACE|ALTER))</string>
+							<key>end</key>
+							<string>(?=''')</string>
+							<key>patterns</key>
+							<array>
+								<dict>
+									<key>include</key>
+									<string>#constant_placeholder</string>
+								</dict>
+								<dict>
+									<key>include</key>
+									<string>#escaped_unicode_char</string>
+								</dict>
+								<dict>
+									<key>include</key>
+									<string>#escaped_char</string>
+								</dict>
+								<dict>
+									<key>include</key>
+									<string>source.sql</string>
+								</dict>
+							</array>
+						</dict>
+						<dict>
+							<key>begin</key>
+							<string>(?=\S)</string>
+							<key>end</key>
+							<string>(?=''')</string>
+							<key>patterns</key>
+							<array>
+								<dict>
+									<key>include</key>
+									<string>#constant_placeholder</string>
+								</dict>
+								<dict>
+									<key>include</key>
+									<string>#escaped_unicode_char</string>
+								</dict>
+								<dict>
+									<key>include</key>
+									<string>#escaped_char</string>
+								</dict>
+							</array>
+						</dict>
+					</array>
+				</dict>
+				<dict>
+					<key>begin</key>
+					<string>(')</string>
+					<key>beginCaptures</key>
+					<dict>
+						<key>1</key>
+						<dict>
+							<key>name</key>
+							<string>punctuation.definition.string.begin.pysa</string>
+						</dict>
+					</dict>
+					<key>comment</key>
+					<string>single quoted string</string>
+					<key>end</key>
+					<string>(')|(\n)</string>
+					<key>endCaptures</key>
+					<dict>
+						<key>1</key>
+						<dict>
+							<key>name</key>
+							<string>punctuation.definition.string.end.pysa</string>
+						</dict>
+						<key>2</key>
+						<dict>
+							<key>name</key>
+							<string>invalid.illegal.unclosed-string.pysa</string>
+						</dict>
+					</dict>
+					<key>name</key>
+					<string>string.quoted.single.single-line.pysa</string>
+					<key>patterns</key>
+					<array>
+						<dict>
+							<key>include</key>
+							<string>#constant_placeholder</string>
+						</dict>
+						<dict>
+							<key>include</key>
+							<string>#escaped_unicode_char</string>
+						</dict>
+						<dict>
+							<key>include</key>
+							<string>#escaped_char</string>
+						</dict>
+					</array>
+				</dict>
+			</array>
+		</dict>
+		<key>strings</key>
+		<dict>
+			<key>patterns</key>
+			<array>
+				<dict>
+					<key>include</key>
+					<string>#string_quoted_double</string>
+				</dict>
+				<dict>
+					<key>include</key>
+					<string>#string_quoted_single</string>
+				</dict>
+			</array>
+		</dict>
+	</dict>
+	<key>scopeName</key>
+	<string>source.pysa</string>
+	<key>uuid</key>
+	<string>F23DB5B2-7D08-11D9-A709-000D93B6E43C</string>
```

Comment:
Does this UUID need to be regenerated? I have no idea where it's used, but I can imagine two files having the same UUID is a bad idea

Reply:
You're right - I just generated a new UUID for each language
![image](https://user-images.githubusercontent.com/38268331/118565372-0c795400-b740-11eb-8fac-525b5ffa6f04.png)